beta-src/src/components/map/components/WDProvince.tsx

```diff
@@ -0,0 +1,136 @@
+import { useTheme } from "@mui/material";
```

Comment:
Can you `git mv` this from `WDTerritory.tsx` so that changes are tracked?

Reply:
No, because git doesn't track renames and so "git mv" has no difference from doing the rename yourself. https://git.wiki.kernel.org/index.php/GitFaq#Why_does_Git_not_.22track.22_renames.3F

You might be thinking of Mercurial or Subversion, which do have explicit rename tracking.

The only thing git does have is heuristic after-the-fact tracking via file similarity. Which you can see works fine, if you git show the relevant individual commit you can see that git caught the rename perfectly well:
```
git show 62e78f9c00b2386f2806532b5bbfe92c5855a86b
...
 beta-src/src/components/map/components/{WDTerritory.tsx => WDProvince.tsx} | 50 +++++++++++++++++++++++++-------------------------
 beta-src/src/components/map/variants/classic/components/WDBoardMap.tsx     | 10 +++++-----
```
However, Git*hub* has no support for showing renames in diffs, even when git's heuristics would detect it fine. So that's what you're seeing.